grails-app/services/org/pih/warehouse/inboundSortation/PutawayResult.groovy

```diff
@@ -0,0 +1,14 @@
+package org.pih.warehouse.inboundSortation
+
+import org.pih.warehouse.core.Location
+import org.pih.warehouse.inventory.InventoryItem
+import org.pih.warehouse.product.Product
+
+class PutawayResult {
+    Location facility
+    Product product
+    InventoryItem inventoryItem
+    Location currentBinLocation
+    Location putawayLocation
```

Comment:
I just noticed that the previous two fields use the old. Let's match the name used in the PutawayTask to make the mapping more straightforward. I might eventually rename these fields a bit. We need container, even though we don't use it now, because eventually we'll try to guess which container should be used.
```
Location location
Location container
Location destination 
```